Jenkinsfile

```diff
@@ -95,6 +96,19 @@ pipeline {
                                     sh script: 'cat .build/tests.json | tparse -all -top -slow 100', returnStatus: true
                                     archiveArtifacts artifacts: '.build/tests.json'
                                     junit '.build/tests.xml'
+
+                                    script {
+                                        if(fileExists(".build/coverprofile")){
+                                            sh script: 'filter-cover-profile < .build/coverprofile > .build/clean.coverprofile', returnStatus: true
+                                            sh script: 'gocov convert .build/clean.coverprofile > .build/cover.json', returnStatus: true
+                                            sh script: 'gocov-xml  < .build/cover.json > .build/cobertura.xml', returnStatus: true
+                                            cobertura coberturaReportFile: '.build/cobertura.xml',
+                                                lineCoverageTargets: '70, 60, 50',
+                                                autoUpdateHealth: false,
+                                                autoUpdateStability: false,
+                                                failUnhealthy: false
+                                        }
+                                    }
```

Comment:
we'll also need it for testsuite down below, I think

Reply:
@amwolff added